pulse_actions/handlers/treeherder_add_new_jobs.py

```diff
@@ -26,7 +26,12 @@ def on_runnable_job_event(data, message, dry_run, treeherder_host, acknowledge):
     repo_name = data["project"]
     requester = data["requester"]
     resultset_id = data["resultset_id"]
-    buildernames = data["buildernames"]
+    if "buildernames" in data:
+        buildernames = data["buildernames"]
+    elif "requested_jobs" in data:
+        buildernames = data["requested_jobs"]
+    else:
+        buildernames = []
```

Comment:
Could you please raise an exception? or LOG.error() with return -1?

I want to be alerted in such cases so I can investigate.